src/test/java/com/spotify/reaper/acceptance/BasicSteps.java

```diff
@@ -299,4 +456,57 @@ public void a_new_daily_repair_schedule_is_added_that_already_exists_for(String
         ReaperTestJettyRunner.callReaper("POST", "/repair_schedule", Optional.of(params));
     assertEquals(Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), response.getStatus());
   }
+ 
+ @And("^the last added repair is activated$")
+ public void the_last_added_repair_is_activated_for()
+     throws Throwable {
+   Map<String, String> params = Maps.newHashMap();
+   ClientResponse response =
+       ReaperTestJettyRunner.callReaper("PUT", "/repair_run/" + TestContext.LAST_MODIFIED_ID + "?state=RUNNING", Optional.of(params));
+   assertEquals(Response.Status.OK.getStatusCode(), response.getStatus());
+   String responseData = response.getEntity(String.class);
+   RepairRunStatus run = SimpleReaperClient.parseRepairRunStatusJSON(responseData);
+   TestContext.LAST_MODIFIED_ID = run.getId();
+ }
+ 
+ @When("^the last added repair is stopped$")
+ public void the_last_added_repair_is_stopped_for()
+     throws Throwable {
+   Map<String, String> params = Maps.newHashMap();
+   ClientResponse response =
+       ReaperTestJettyRunner.callReaper("PUT", "/repair_run/" + TestContext.LAST_MODIFIED_ID + "?state=PAUSED", Optional.of(params));
+   assertEquals(Response.Status.OK.getStatusCode(), response.getStatus());
+   String responseData = response.getEntity(String.class);
+   RepairRunStatus run = SimpleReaperClient.parseRepairRunStatusJSON(responseData);
+   TestContext.LAST_MODIFIED_ID = run.getId();
+ }
+ 
+ @And("^we wait for (\\d+) seconds$")
+ public void we_wait_for(int sleepTime)
+     throws Throwable {
+   LOG.info("Sleeping for " + sleepTime + "s...");
+   Thread.sleep(sleepTime * 1000);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this use of "Thread.sleep()". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2925)